src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -374,40 +326,37 @@ public ActivityProfileState isActive(final AlignmentContext context, final Refer
-            return new ActivityProfileState(ref.getInterval(), 0);
+            return new ActivityProfileState(refInterval, 0);
         }
 
-        final ReadPileup tumorPileup = tumorContext.getBasePileup().makeFilteredPileup(el -> el.getMappingQual() >= MTAC.MIN_MAPPING_QUALITY_SCORE);
-        final double[] tumorGLs = calcGenotypeLikelihoodsOfRefVsAny(tumorPileup, ref.getBase(), MTAC.minBaseQualityScore);
-        final double tumorLod = tumorGLs[1] - tumorGLs[0];
-
-        // NOTE: do I want to convert to a probability (or just keep this as a LOD score)
-
-        // also at this point, we should throw out noisy sites (hence the nonRefInNormalCheck) but this is non-optimal
-        double prob = 0;
-        if (tumorLod > MTAC.INITIAL_TUMOR_LOD_THRESHOLD) {
-
-            // TODO: should we even do this performance optimization?
-            // in any case, we have to handle the case where there is no normal (and thus no normal context) which is
-            // different than having a normal but having no reads (where we should not enter the active region)
-            // TODO: as mentioned above, when we provide the normal bam but there are no normal reads in the region, we call it active. This does not seem right to me.
-            if (hasNormal() && normalContext != null) {
-                final int nonRefInNormal = getCountOfNonRefEvents(normalContext.getBasePileup(), ref.getBase(), MTAC.minBaseQualityScore);
-
-                final double[] normalGLs = calcGenotypeLikelihoodsOfRefVsAny(normalContext.getBasePileup(), ref.getBase(), MTAC.minBaseQualityScore, 0.5f);
-                final double normalLod = normalGLs[0] - normalGLs[1];
-
-                // TODO: parameterize these
-                if (normalLod > 1.0 && nonRefInNormal < 4) {
-                    prob = 1;
-                    logger.debug("At " + ref.getInterval().toString() + " tlod: " + tumorLod + " nlod: " + normalLod + " with normal non-ref of " + nonRefInNormal);
-                }
-            } else {
-                prob = 1;
-                logger.debug("At " + ref.getInterval().toString() + " tlod: " + tumorLod + " and no-normal calling");
-            }
+        final int tumorNonRef = countNonRef(refBase, tumorContext);
+        if (tumorNonRef < MTAC.minVariantsInPileup) {
+            return new ActivityProfileState(refInterval, 0.0);
+        }
+
+        // since errors are rare, the number of errors (if reads are independent) is approximately a Poisson random variable,
+        // with mean equal to its variance
+        final double expectedTumorNonRefDueToError = StreamSupport.stream(tumorContext.getBasePileup().spliterator(), false)
+                .mapToDouble(pe -> QualityUtils.qualToErrorProb(pe.getQual()))
+                .sum();
+        final double tumorNonRefStdev = Math.sqrt(expectedTumorNonRefDueToError);
+
+        if (tumorNonRef < expectedTumorNonRefDueToError + MTAC.tumorStandardDeviationsThreshold * tumorNonRefStdev) {
```

Comment:
This is as intended -- it's saying that we need to be a few standard deviations above the expected number of alt counts to be called active.  Although you could always set `MTAC.tumorStandardDeviationsThreshold` to be negative. . .